### DISCUSSION TITLE
How to change the color of the outer section of an Angular gauge?

### DISCUSSION THREAD

**Discussion by KomodoCrypto** [Live-Charts/LiveCharts2#1460] [General]
I can change the color of the series, but how to change the default color of the outer section that is, as default, gray? 
![image](https://github.com/beto-rodriguez/LiveCharts2/assets/973309/4ee71327-22f6-480c-8e1a-b4a8489be76c)

**Comment by N4gl3r** [comment]
Here is a code snippet I used... 

GaugeItem normalRangeItem = CreateNormalRangeItem(needle);
GaugeItem warningRangeItem = CreateWarningRangeItem(needle);
GaugeItem criticalRangeItem = CreateCriticalRangeItem(needle);

GaugeItem backgroundItem = new GaugeItem(GaugeItem.Background, series =>
{
    series.Name = "Background";
    series.Fill = null; // with color --> series.Fill = new SolidColorPaint(SKColors.DodgerBlue);
});

GaugeItem[] gaugeItems = new GaugeItem[]
{
    normalRangeItem,
    warningRangeItem,
    criticalRangeItem,
    backgroundItem,
};

needle.Series = GaugeGenerator.BuildAngularGaugeSections(gaugeItems);

Hope that helps.

**Comment by NikoLu313** [comment]
private static void SetStyle(double sectionsOuter, double sectionsWidth, SKColor color, PieSeries<ObservableValue> series)
 {
     series.OuterRadiusOffset = sectionsOuter;
     series.MaxRadialColumnWidth = sectionsWidth;
     **_series.Fill = new SolidColorPaint(color);_**
 }

**Comment by toshiVon** [comment]
How are you able to change the colors of the inner?

**Comment by N4gl3r** [comment]
Not sure what you mean with the 'inner section'.
But the gauge generator needs an array of gauge items. Each gauge item defines a colored range.

```
GaugeItem[] gaugeItems = new GaugeItem[]
{
normalRangeItem,
warningRangeItem,
criticalRangeItem,
backgroundItem,
};
needle.Series = GaugeGenerator.BuildAngularGaugeSections(gaugeItems)
```

You can set the color for a gauge item in a property of the associated series in the following way.

```
GaugeItem normalRangeItem = new(needle.WarningValue, series =>
{
    series.Name = _normalValueRangeName;
    series.OuterRadiusOffset = 75;
    series.MaxRadialColumnWidth = needle.Thickness;
    series.Fill = new SolidColorPaint(SKColors.DodgerBlue);
})

GaugeItem warningRangeItem = new(warningRangeValue, series =>
{
    series.Name = _warningValueRangeName;
    series.OuterRadiusOffset = 75;
    series.MaxRadialColumnWidth = needle.Thickness;
    series.Fill = new SolidColorPaint(SKColors.Orange);
})

GaugeItem criticalRangeItem = new(criticalRangeValue, series =>
{
    series.Name = _criticalValueRangeName;
    series.OuterRadiusOffset = 75;
    series.MaxRadialColumnWidth = needle.Thickness;
    series.Fill = new SolidColorPaint(SKColors.Red);
})
```

Others are welcome to correct me.

**Reply by toshiVon** [reply]
Sorry, i meant to change the gauge item's color. But i also did it by writing the same pattern of code. Thanks for the response!
